src/App.jsx

```diff
@@ -1,10 +1,27 @@
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import Home from './views/Home/Home';
+import Login from './views/Login/Login';
 import LayoutViews from './views/LayoutViews/LayoutViews';
+import { PrivateRoute } from './components/PrivateRoute/PrivateRoute';
+import { ProvideAuth } from './context/ProvideAuth';
 
 export default function App() {
   return (
-    <LayoutViews>
-      <Home />
-    </LayoutViews>
+    <>
+      <ProvideAuth>
+        <Router>
+          <Switch>
+            <Route exact path="/login">
+              <Login />
```

Comment:
Opening route is straight to login then once you login it will send you to your home page which is a protected route